Src/ScipBe.Common.Office/ScipBe.Common.Office.nuspec

```diff
@@ -2,7 +2,7 @@
 <package>
   <metadata>
     <id>ScipBe.Common.Office</id>
-    <version>2.0.2</version>
+    <version>2.0.3</version>
```

Comment:
Lets change all versions to 3.0. All your great changes make it ready to use with .NET Core which is a huge improvement